db/findGrapherReferencesInSDGTracker.ts

```diff
@@ -0,0 +1,110 @@
+import fs from "fs-extra"
+import path from "path"
+import fetch from "node-fetch"
+import uniq from "lodash/uniq"
+
+import * as db from "./db.js"
+
+const SDG_TRACKER_PAGE_URLS: string[] = [
+    "https://sdg-tracker.org/no-poverty",
```

Comment:
Do we want to do this with a locally-cloned `sdg-tracker.org` repo instead, which will exist as a sibling to `owid-grapher` similar to how we do it for `owid-content`?

Reply:
It seems most robust to do it on the live site?

Otherwise we have to worry about pulling the repo (and things that can go wrong with that), being on a different branch, and additional manual setup for it to work.